src/System.IO.Compression.Brotli/README.md

```diff
@@ -0,0 +1,214 @@
+﻿# System.IO.Compression.Brotli
+
+## Introduction
+
+Brotli is a generic-purpose lossless compression algorithm that compresses data
+using a combination of a modern variant of the LZ77 algorithm, Huffman coding
+and 2nd order context modeling, with a compression ratio comparable to the best
+currently available general-purpose compression methods. It is similar in speed
+to deflate but offers more dense compression.
+
+The specification of the Brotli Compressed Data Format is defined in [RFC 7932](https://www.ietf.org/rfc/rfc7932.txt).
+
+Brotli encoding is supported by most web browsers, major web servers, and some CDNs.
+
+## Brotli Class
+```C#
+public static class Brotli {
+        public static TransformationStatus Compress(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, ref Brotli.State state);
+        public static TransformationStatus Decompress(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, ref Brotli.State state);
+        public static TransformationStatus FlushEncoder(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, ref Brotli.State state, bool isFinished=true);
+        public static int GetMaximumCompressedSize(int inputSize);
+        public struct State : IDisposable {
+            public bool CompressMode { get; }
+            public void Dispose();
+            public void SetQuality(uint quality);
+            public void SetWindow(uint window);
+        }
+    }
+```
+```out bytesConsumed``` - number of bytes used from source data while executing the operation
+
+```out bytesWritten``` - number of bytes are written in destination while executing the operation
+
+```State``` is used in both compress and decompress mode to save a temporary status and data of process.  State will set automaticaly at first call of Compress or Decompress. Once it is initialized and being used for one data stream, it shouldn't be used for another one and in other mode.
```

Comment:
* automaticaly => automatically
* "at first call" => "on initial call"
* Remove sentence "Once it is initialized..." and replace with "The state cannot be re-used for multiple data streams and cannot switch modes (compress vs. decompress)."